src/components/login/LoginForm.js

```diff
@@ -0,0 +1,96 @@
+import { useState } from "react";
+import { FaTwitter } from "react-icons/fa";
+import { axiosInstanceLog } from "../api/axios";
+import "./LoginForm.css";
+
+const LoginForm = ({ setAuthUser, closeModal }) => {
```

Comment:
Where are you getting `closeModal` from?

Reply:
am getting it from the `signUp` compenent

Reply:
my main goal is to toggle between the `LoginForm` & `signUp` component on the `App` component. but presently I can only add one component to login to the `authUser` state in the `App` else it will login but will not proceed to the `authuser` state in the `App` component. Can I achieve this method without using `UserContext` in react?